### ISSUE TITLE
L'elan (bmtnaaf) 1915-06-01_01 Excess related items

### ISSUE THREAD

**Issue by tanvisol** [pulibrary/BlueMountain#59]
Related item 18-20 should be deleted. 18 and 20 and 19 is not its own text but part of Poussieres.

http://bluemountain.princeton.edu/bluemtn/cgi-bin/bluemtn?a=d&d=bmtnaaf19150601-01&e=-------en-20--1--txt-txIN-------#

**Comment by cwulfman** [comment]
I don't think we should create separate sub-related items for each of these sub-sections of Poussieres.  So the separate MODS related items should be deleted, and, in the structMap, the divs should be consolidated into a single block under the heading Poussieres.

**Comment by JNG-blue** [comment]
ok deleting everything except poissieres and the tiny illustration and throwing them all into possieres

**Comment by JNG-blue** [comment]
All heads made into subheads and put into copy/bodycontent for c014. check carefully to make sure the subheads came out in the right order.  make sure to check on the dev server, above link is not to the dev server.
